README.md

```diff
@@ -36,7 +36,7 @@ graphics, video, text, and controls without limitation. Flutter includes a full
 [set of widgets][widget catalog] that deliver pixel-perfect experiences on both
 iOS and Android.
 
-![Reflectly hero image][Reflectly hero image]
```

Comment:
Is it still possible to use the reference to `Reflectly hero image` in the `src` attribute?  Maybe not.

Reply:
No, we can't use it in that way